analytics/terraform/datahub-on-eks/vpc.tf

```diff
@@ -34,10 +34,10 @@ module "vpc" {
 }
 
 module "vpc_endpoints_sg" {
-  count = var.enable_vpc_endpoints ? 1 : 0
```

Comment:
all of our modules have a `create` logic built in, usually under the variable `create`. Its better to use this variable for conditional creation since has better support for the disabled condition in its outputs